packages/reactDom/test/test_RSC_model.ml

```diff
@@ -370,7 +374,10 @@ let suspense_with_immediate_promise () =
   let output, subscribe = capture_stream () in
   let%lwt () = ReactServerDOM.render_model ~subscribe main in
   assert_list_of_strings !output
-    [ "0:[\"$\",\"$Sreact.suspense\",null,{\"fallback\":\"Loading...\",\"children\":\"DONE :)\"},null,[],{}]\n" ];
+    [
+      "1:\"DONE :)\"\n";
+      "0:[\"$\",\"$Sreact.suspense\",null,{\"fallback\":\"Loading...\",\"children\":\"$L1\"},null,[],{}]\n";
```

Comment:
This is conceptually incorrect. How's it possible that the resolved promise comes before the suspense node?